src/Layout/redist/targets/GenerateLayout.targets

```diff
@@ -30,6 +30,22 @@
     <Delete Files="@(RoslynFrameworkAssemblies)" />
   </Target>
 
+  <Target Name="PublishNETAnalyzers"
+          BeforeTargets="Build;PublishNetSdk">
+     <PropertyGroup>
+      <AnalyzerAssembliesDirectory>$(ArtifactsBinDir)$(Configuration)\Sdks\Microsoft.NET.Sdk\analyzers</AnalyzerAssembliesDirectory>
+      <AnalyzerTargetsDirectory>$(AnalyzerAssembliesDirectory)\build</AnalyzerTargetsDirectory>
+    </PropertyGroup>
+    <ItemGroup>
+      <AnalyzerAssemblies Include="$(NuGetPackageRoot)/microsoft.codeanalysis.netanalyzers/$(MicrosoftCodeAnalysisNetAnalyzersVersion)/analyzers/dotnet/**/*" />
+      <AnalyzerTargets Include="$(NuGetPackageRoot)/microsoft.codeanalysis.netanalyzers/$(MicrosoftCodeAnalysisNetAnalyzersVersion)/build/**/*" />
+    </ItemGroup>
+    <Error Condition="'@(AnalyzerAssemblies)' == ''" Text="Something moved around in Analyzer package, adjust code here accordingly. TFM change?" />
```

Comment:
Please use `NETSdkError` here instead for localized error messages

Reply:
This is part of the build of the SDK, not something we ship, so we don't need to (and shouldn't) add localization for this.